src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages_tr.properties

```diff
@@ -9,6 +9,7 @@ declaration.order.constructor=''constructor'' tanımı yanlış sırada yapılm
 declaration.order.instance=Değişken tanımı yanlış sırada yapılmış.
 declaration.order.static=''static'' değişken tanımı yanlış sırada yapılmış.
 default.comes.last=''switch'' içerisindeki ''default'' ifadesi son durum olarak yer almalıdır.
+diamond.operator.for.variable.definition=Boş bir tür parametre kümesi kullanın.
```

Comment:
This is the culprit.
Messages aren't very clear in TC, but if you expand them all `messages_tr.properties` says the unique message:
> 1: Properties file is alphabetically unsorted